default/generated/spring-framework/spring-framework-5.x-to-6.0-data-access.yaml

```diff
@@ -29,6 +29,71 @@
   - title: 'Spring 6.0 migration guide'
     url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#removed-apis
 
+- ruleID: spring-framework-5.x-to-6.0-data-access-00002
+  category: mandatory
+  effort: 1
+  labels:
+  - konveyor.io/source=spring5
+  - konveyor.io/target=spring6+
+  - konveyor.io/target=hibernate6+
+  when:
+    or:
+    - builtin.filecontent:
+        filePattern: (hibernate\.properties|persistence\.xml|cfg\.xml|application\.properties|application\.yaml)
+        pattern: hibernate.bytecode.provider=javassist
+    - as: hibernate_cfg
```

Comment:
I think we can remove this `as` field, since it's now being used anywhere. The same with the one below, `as: persistence_files`.

Reply:
Removed